libs/ActivityKit/include/activities/SuperSimon.h

```diff
@@ -0,0 +1,66 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+// LCOV_EXCL_START
+
+#include <random>
+
+#include "Color.h"
+#include "MathUtils.h"
+#include "Number.h"
+#include "RFIDKit.h"
+#include "ReinforcerKit.h"
+#include "interface/Activity.h"
+#include "interface/libs/VideoKit.h"
+
+namespace leka::activity {
+
+class SuperSimon : public interface::Activity
+{
+  public:
+	explicit SuperSimon(RFIDKit &rfidkit, interface::VideoKit &videokit, ReinforcerKit &reinforcerkit,
+						interface::LED &led)
+		: _rfidkit(rfidkit), _videokit(videokit), _reinforcerkit(reinforcerkit), _led(led) {};
+
+	void start() final;
+	void stop() final;
+
+  private:
+	RFIDKit &_rfidkit;
+	interface::VideoKit &_videokit;
+	ReinforcerKit &_reinforcerkit;
+	interface::LED &_led;
+
+	static constexpr uint8_t kNumberOfColors = 6;
+	static constexpr uint8_t kScoreMax		 = 10;
+	static constexpr uint8_t kNumberOfLeds	 = 20;
+
+	static constexpr std::array<Number, 10> numbers_table = {
+		Number::number_1, Number::number_2, Number::number_3, Number::number_4, Number::number_5,
+		Number::number_6, Number::number_7, Number::number_8, Number::number_9, Number::number_10};
+
+	uint8_t _score		  = 0;
+	uint8_t _pos		  = 0;
+	uint8_t _index_number = 0;
+	uint8_t _score_to_win = 5;
```

Comment:
```suggestion
	uint8_t _current_round		  = 0;
	uint8_t _pos		  = 0;
	uint8_t _index_number = 0;
	uint8_t _number_of_rounds = 5;
```

Reply:
what is `_pos` and `_index_number`?

position and index are usually the exact same thing